src/styles/styles.css

```diff
@@ -82,3 +126,63 @@
     font-size: 1rem;
     padding: 0.25em 0.5em;
 }
+
+/* ********  GAME-OVER-PAGE  ******** */
+
+#game-over-page {
+    /* visibility: hidden; */
+}
+
+#game-over-title {
+    margin-bottom: 1.5rem;
+}
+
+/* ********  COMMON STYLES  ******** */
+
+#start-page,
```

Comment:
I think the different menus should only cover the #game-page, not the entire viewport.